### ISSUE TITLE
Cannot advance to the next page feature

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#113]
Screengrab/videos (insert file names here)
2020 04 24, Cannot advance to the next page feature
[2020 04 24 Cannot advance to the next page feature.mp4.zip](https://github.com/CityOfBoston/Iterators/files/4384334/2020.04.24.Cannot.advance.to.the.next.page.feature.mp4.zip)
.mp4

Link
https://d8-stg.boston.gov/news

Node # (if known)
1401

Type of User:  Anonymous

Environment
(Device > OS > Browser)

Computer > MacOS 10.15.3 > Chrome 80
Trusted Tester Test
4) Keyboard and Focus

Brief Description
Cannot advance to the next page feature on Node 1401

Steps to reproduce
1.	00:00, Go to https://d8-stg.boston.gov/news
2.	00:06, Start tabbing through site with the keyboard only.
3.	00:46, Tabbing (you cannot see this in the video) onto the last posting on the page titled, Demolition, Delay Application 46…and continuing to tab. There is an invisible tab order as cursor cannot be found.
a.	This is a bug as user should know at all times where the location of the cursor is. Also, tabbing to the number field below is the only logical place the cursor should go. 
4.	00:54, Tabbing and using the Shift Tab feature to move cursor in attempt to land on the numbered fields below unsuccessful. 

Desired Result
•	Able to tab through the page successfully and navigate the advance to next page feature.

Actual Result
•	Unable to tab through the page successfully and navigate the advance to next page feature.
